supervisor/shared/usb/usb.c

```diff
@@ -87,11 +92,13 @@ void usb_background(void) {
 
 // Invoked when device is mounted
 void tud_mount_cb(void) {
+    g_usb_msc_enabled = true;
```

Comment:
I don't think this needed. `mount` doesn't mean filesystem mount here unfortunately. It is plugged/unplugged.